src/v2/Components/Address/AddressModalFields.tsx

```diff
@@ -1,10 +1,36 @@
 import React from "react"
-import { Box, Column, Flex, GridColumns, Input, Text } from "@artsy/palette"
+import {
+  Box,
+  Column,
+  Flex,
+  GridColumns,
+  Input,
+  Text,
+  useThemeConfig,
+} from "@artsy/palette"
 import { useFormikContext } from "formik"
 import { SavedAddressType } from "v2/Apps/Order/Utils/shippingUtils"
 import { CountrySelect } from "../CountrySelect"
+import styled from "styled-components"
+
+const AllCapsText = styled(Text)`
+  text-transform: uppercase;
+`
 
 export const AddressModalFields: React.FC = () => {
+  const elements = useThemeConfig({
+    v2: {
+      fieldLabel: <Text mb={0.5}>Country</Text>,
+    },
+    v3: {
+      fieldLabel: (
+        <AllCapsText fontSize="13px" mb={0.5}>
```

Comment:
Why not use `<Text>`?

Reply:
That & use a variant. Never set a fontSize directly.

Reply:
👍 Didn't realize we had a variant for 13px